src/features/minimap/minimapSlice.ts

```diff
@@ -42,5 +45,6 @@ export const setShowMinimapAction =
 export const showMinimap = (state: RootState) => state.app.minimap.showMinimap;
 export const getXAxisMaxTime = (state: RootState) =>
     state.app.minimap.xAxisMaxTime;
-export const { setShowMinimap, miniMapAnimationAction } = minimapSlice.actions;
+export const { setShowMinimap, miniMapAnimationAction, resetMinMap } =
+    minimapSlice.actions;
```

Comment:
For these as well, it's minimap and not mini map, so that we write `minimap` and not `miniMap` :)